src/calculateRentalCost.js

```diff
@@ -4,7 +4,19 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const Price = 40;
+  const PriceForWeek = 50;
+  const PriceForThreeDays = 20;
+
+  if (days >= 7) {
+    return days * Price - PriceForWeek;
```

Comment:
Use a constant to represent the number of days that qualify for a long-term rental discount instead of using the literal `7` directly in the expression. This will make the code more readable and maintainable.